### PR TITLE
attributes/sysconfig.rb - added `--` to $DAEMON_OPTS

### PR DESCRIPTION
**Pull request by ericfrancis**
Tested on ubuntu 14.04

This fixes an issue where `start-stop-daemon` could not distinguish the `--config` argument passed to its `--exec` argument

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,9 @@ Berksfile.lock
 Gemfile.lock
 
 metadata.json
+
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion
+*.iml
+## Directory-based project format:
+.idea/
```
Comment (triccardi-systran) [inline .gitignore]: Ideally this should be in a separate PR.

**Comment by shortdudey123** [comment]
Does not appear this attribute is even used and will probably be removed in a future release

```
$ date
Fri Dec  9 00:00:39 PST 2016
$ git grep DAEMON_OPTS
attributes/sysconfig.rb:default['mongodb']['sysconfig']['DAEMON_OPTS'] = "--config #{node['mongodb']['dbconfig_file']}"
templates/default/debian-mongodb.init.erb:    DAEMON_OPTS="$DAEMON_OPTS"
templates/default/debian-mongodb.init.erb:    DAEMON_OPTS="-- $DAEMON_OPTS"
templates/default/debian-mongodb.init.erb:echo "** Running $NAME ($DAEMON $DAEMON_OPTS)"
templates/default/debian-mongodb.init.erb:                        --exec $NUMACTL $DAEMON $DAEMON_OPTS
templates/default/debian-mongodb.upstart.erb:      DAEMON_OPTS="$DAEMON_OPTS"
templates/default/debian-mongodb.upstart.erb:      DAEMON_OPTS="-- $DAEMON_OPTS"
templates/default/debian-mongodb.upstart.erb:  exec start-stop-daemon --start --quiet --chuid $DAEMON_USER --exec $NUMACTL $DAEMON $DAEMON_OPTS
templates/default/redhat-mongodb.init.erb:  daemon --user "$DAEMON_USER" $NUMACTL $DAEMON $DAEMON_OPTS
$ 

```
